src/test/java/org/dependencytrack/resources/v1/ProjectResourceTest.java

```diff
@@ -348,7 +348,7 @@ public void getProjectsConciseWithAclTest() {
                     "hasChildren": false
                   }
                 ]
-                """);
+                """.formatted(team.getName()));
```

Comment:
You still need to extend the JSON to include the `teams` field, just like you did here: https://github.com/DependencyTrack/hyades-apiserver/pull/855/files#diff-24f2f9ee52fee0de1bc5c70446b58dd249f9e0c0c6a24514d5dddf028b010589R1142-R1146

Reply:
Thank for you pointing that out